packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -369,6 +369,64 @@ function QueueFromGitOps() {
     );
   }
 
+  const disablePreviewButton =
+    chainId !== currentSafe?.chainId ||
+    settings.isIpfsGateway ||
+    !cannonDefInfo.def ||
+    cannonPkgPreviousInfo.isFetching ||
+    partialDeployInfo.isFetching ||
+    cannonPkgVersionInfo.isFetching ||
+    buildInfo.isBuilding;
+
+  function PreviewButton(props: any) {
+    return (
+      <Tooltip label={props.message}>
+        <Button
+          width="100%"
+          colorScheme="teal"
+          isDisabled={disablePreviewButton}
+          onClick={() => buildInfo.doBuild()}
+        >
+          Preview Transactions to Queue
+        </Button>
+      </Tooltip>
+    );
+  }
+
+  function RenderPreviewButtonTooltip() {
+    if (settings.isIpfsGateway) {
+      return (
+        <PreviewButton message="Cannot write to IPFS endpoint as gateway, please change the IPFS Url in your settings." />
+      );
+    }
```

Comment:
I think all the logic for `isIpfsGateway` can be removed, as it always returns `false` now: https://github.com/usecannon/cannon/blame/main/packages/builder/src/ipfs.ts/#L44-L47

Reply:
Oh right! Ok I'll remove this :)